browser/brave_news/brave_news_tab_helper.cc

```diff
@@ -48,13 +48,19 @@ BraveNewsTabHelper::GetAvailableFeeds() {
   std::vector<FeedDetails> feeds;
 
   base::flat_set<GURL> seen_feeds;
-  auto* default_publisher =
-      controller_->publisher_controller()->GetPublisherForSite(
-          GetWebContents().GetLastCommittedURL());
-  if (default_publisher) {
-    seen_feeds.insert(default_publisher->feed_source);
-    feeds.push_back(
-        {default_publisher->feed_source, default_publisher->publisher_name});
+
+  auto current_url = GetWebContents().GetLastCommittedURL();
+
```

Comment:
nit/totally optional: Removing this line could be more readable as the variable and if () statement is grouped together.

```suggestion
```